client/src/main/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClient.java

```diff
@@ -35,21 +35,31 @@
   private final Map<String, Map<Schema, Integer>> schemaCache;
   private final Map<Integer, Schema> idCache;
   private final Map<String, Map<Schema, Integer>> versionCache;
+  private boolean enableAutoSchemaRegistry = true;
 
   public CachedSchemaRegistryClient(String baseUrl, int identityMapCapacity) {
-    this(new RestService(baseUrl), identityMapCapacity);
+    this(new RestService(baseUrl), identityMapCapacity, true);
```

Comment:
It would be better not to have `enableAutoSchemaRegistry` in CachedSchemaRegistryClient - just leave this in AbstractKafkaAvroSerializer